tests/maintel/test_mtcs.py

```diff
@@ -0,0 +1,178 @@
+                        state = await self.mtcs.get_state(comp)
+                        self.assertEqual(state, salobj.State.ENABLED)
+
+            # Send all components to STANDBY state
+            await self.mtcs.standby()
+
+            await asyncio.sleep(HB_TIMEOUT)
+
+            #  Check that all components are in STANDBY state
+            #  Check that all components are in STANDBY state
+            for comp in self.mtcs.components:
+                if comp not in self.mtcs_mock.output_only:
+                    with self.subTest("check final state", component=comp):
+                        state = await self.mtcs.get_state(comp)
+                        self.assertEqual(state, salobj.State.STANDBY)
+
+    async def test_point_azel(self):
+
+        async with self.make_group(
+            usage=MTCSUsages.StartUp + MTCSUsages.Shutdown + MTCSUsages.Slew
+        ):
+
+            # enable all components
+
+            await self.mtcs.enable()
+
+            # slew telescope and rotator to random position,
+            # do not wait for the dome
+            # azimuth, random values between +/- 90.
+            az_set = ((np.random.random() - 0.5) * 2.0) * 90.0
+            # elevation, random values between 10 and 80.
+            el_set = np.random.random() * 70.0 + 10.0
+            # rotator, rando value between +/- 90.
+            rot_set = ((np.random.random() - 0.5) * 2.0) * 90.0
+
+            ret_val = await self.mtcs.point_azel(
+                az=az_set,
+                el=el_set,
+                rot_tel=rot_set,
+                target_name="random",
+                wait_dome=False,
+                slew_timeout=SLEW_TIMEOUT,
+            )
+
+            print(f"Point azel returned: {ret_val}")
+
+            az_rec = await self.mtcs.rem.mtmount.tel_Azimuth.next(
+                flush=True, timeout=HB_TIMEOUT
+            )
+
+            el_rec = await self.mtcs.rem.mtmount.tel_Elevation.next(
+                flush=True, timeout=HB_TIMEOUT
+            )
+
+            rot_rec = await self.mtcs.rem.rotator.tel_Application.next(
+                flush=True, timeout=HB_TIMEOUT
+            )
+
+            self.assertEqual(az_rec.Azimuth_Angle_Set, az_set)
+            self.assertEqual(el_rec.Elevation_Angle_Set, el_set)
+            self.assertEqual(rot_rec.Demand, rot_set)
+
+    async def test_slew_all(self):
+
+        async with self.make_group(
+            usage=MTCSUsages.StartUp + MTCSUsages.Shutdown + MTCSUsages.Slew
+        ):
+
+            # enable all components
+
+            await self.mtcs.enable()
+
+            # This is a circumpolar object, should be always vizible in
+            # Pachon.
+
+            name = "HD 185975"
+            ra = Angle("20:28:18.74", unit=u.hourangle)
+            dec = Angle("-87:28:19.9", unit=u.deg)
+
+            # Test that slew command works with RA/Dec alone
+            with self.subTest("test_slew", ra=ra, dec=dec):
```

Comment:
I strongly suggest removing `subTest` as used here. It is intended for loops (repeated tests over different values) where it is not clear what the value was. In this case you only have the one case, so a failure will be obvious from the traceback.